src/typescript/sdk/src/const.ts

```diff
@@ -58,6 +58,10 @@ export const BASE_VIRTUAL_CEILING = 49_000_000_000_000_000n;
 export const QUOTE_VIRTUAL_CEILING = 1_400_000_000_000n;
 export const POOL_FEE_RATE_BPS = 25;
 export const MARKET_REGISTRATION_FEE = 100_000_000n;
+export const MARKET_REGISTRATION_DEPOSIT = 400_000_000n;
```

Comment:
```suggestion
export const MARKET_REGISTRATION_DEPOSIT = 4 * ONE_APT_BIGINT;
```
Maybe a bit clearer this way